test/benchmarks/micro/host_adler32.inputs

```diff
@@ -0,0 +1,18 @@
+1
+adler32
```

Comment:
Need to change this name due to issue in wasm3: https://github.com/wasm3/wasm3/issues/143